src/Form/Form.tsx

```diff
@@ -1,9 +1,56 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./Form.css";
 
-const Form = () => {
+export interface Quote {
+  q: string;
+  a: string;
+  h: string;
+}
+
+interface formProps {
+  setAllQuotes: React.Dispatch<React.SetStateAction<Quote[]>>
+}
+
+
+interface Mood{
+  k: string;
+  l: string;
+}
+
+const Form = ({setAllQuotes}: formProps) => {
+  const [moods, setMoods] = useState<Mood[]>([])
+
+  useEffect(() => {
+    fetch("api/keywords?key=11ef57ae8191dde524535934c158c4543950e06c")
+    .then(response => response.json())
+    .then(data => setMoods(data))
+  })
```

Comment:
Can we move this to the api calls file?

Reply:
Yes! Thanks for the reminder. This seems to be the industry stranger, right? We'll be sure to add this to our project board.